lua/lualine/components/filename.lua

```diff
@@ -90,6 +91,10 @@ M.update_status = function(self)
     data = self.options.symbols.unnamed
   end
 
+  if self.options.custom_shortener then
+    data = self.options.custom_shortener(data, path_separator)
+  end
+
   if self.options.shorting_target ~= 0 then
```

Comment:
when custom shorter is used we should disable this.

Reply:
Are you sure? It can be nice to have both features (without user re-implementing `shorten_path`). And it can always be disabled by setting `shorting_target`. And

Reply:
Alright, can be disabled on the user side easily enough. then it's fine.